core/commands/dht.go

```diff
@@ -68,7 +69,9 @@ var queryDhtCmd = &cmds.Command{
 		events := make(chan *notif.QueryEvent)
 		ctx := notif.RegisterForQueryEvents(req.Context(), events)
 
-		closestPeers, err := dht.GetClosestPeers(ctx, key.Key(req.Arguments()[0]))
+		k := string(b58.Decode(req.Arguments()[0]))
```

Comment:
This actually is a bugfix. Previously we we'rent base58 decoding this like we were supposed to.